listenbrainz/mbid_mapping_writer/job_queue.py

```diff
@@ -260,15 +260,17 @@ def update_metrics(self, stats):
                         low_quality_rate=stats["low_quality"] - stats["last_low_quality"],
                         no_match_rate=stats["no_match"] - stats["last_no_match"],
                         listens_per_sec=listens_per_sec,
-                        listen_count=stats["listen_count"],
-                        listens_matched=stats["listens_matched"],
+                        listens_matched_p=stats["listens_matched"] / stats["listen_count"] * 100.0,
```

Comment:
Calculation looks good but don't know what format etc grafana expects this to be in.

Reply:
As per our current discussions in IRC, I find it easier to do the calculations in python than wrestling with the influx query syntax, so calculating what I want to show (%) is just easier.